packages/voter-dapp/src/drizzleOptions.js

```diff
@@ -1,7 +1,13 @@
-import DesignatedVotingFactory from "@uma/core/build/contracts/DesignatedVotingFactory.json";
-import Governor from "@uma/core/build/contracts/Governor.json";
-import Voting from "@uma/core/build/contracts/Voting.json";
-import VotingToken from "@uma/core/build/contracts/VotingToken.json";
+import DesignatedVotingFactory from "@uma/core-1-2-0/build/contracts/DesignatedVotingFactory.json";
```

Comment:
Why can't you use the getAbi(contract, versionString) method here?

Reply:
Not sure any of those methods work in the browser since they require `fs` calls which I think are only available in node.